src/pages/staticPages/visualize/visualize.module.scss

```diff
@@ -4,3 +4,10 @@
         padding-right: 40px;
     }
 }
+
+.customToolArray {
```

Comment:
this class is used by the "3rd party" section - it is the same as toolArray but wider, which looks better here.